src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,130 @@
+import { FC } from "react";
+import { Link, useLocation } from "react-router-dom";
+import {
+  StyledSidebar,
+  StyledHeader,
+  Overlay,
+  CloseButton,
+  HeaderText,
+  ConnectButton,
+  StyledMenu,
+  StyledMenuItem,
+  ConnectText,
+} from "./Sidebar.styles";
+import { onboard, CHAINS } from "utils";
+import { useConnection } from "state/hooks";
+
+const { init, reset } = onboard;
+
+interface Props {
+  openSidebar: boolean;
+  setOpenSidebar: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const Sidebar: FC<Props> = ({ openSidebar, setOpenSidebar }) => {
+  const { account, isConnected, chainId } = useConnection();
+  const location = useLocation();
+
+  return (
+    <>
+      {openSidebar && <Overlay />}
+      <StyledSidebar className={openSidebar ? "open" : ""}>
+        <StyledHeader>
+          <CloseButton onClick={() => setOpenSidebar(false)}>X</CloseButton>
+          <ConnectText isConnected={isConnected}>
+            <div /> {isConnected ? "Connected" : "Disconnected"}
+          </ConnectText>
+          {account && <HeaderText>{account}</HeaderText>}
+          {chainId && isConnected && (
+            <HeaderText>{CHAINS[chainId].name}</HeaderText>
+          )}
+          <ConnectButton
+            onClick={() => {
+              if (isConnected) {
+                return reset();
+              } else {
+                return init();
+              }
+            }}
+          >
```

Comment:
```suggestion
          <ConnectButton onClick={() => isConnected ? reset() : init() }>
```
The ternary operator would save some lines

Reply:
Good call!